share/po/fr.po

```diff
@@ -0,0 +1,83 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
In Engine and CLI the PO files are stored in `share/`, not in `share/po/`. We should have the same model in all respositories when possible.

The PO files could be moved to `share/po/`, but then it should be done in all three respositories. In this repository there are quite a few files already inte `share/`.

Reply:
I've moved them back to `share/`